tools/cldr-apps/src/cldr-root/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
for more consistent html formatting we could add this file to pretty.sh

Reply:
yes, though it shouldn't change very often!  I ran it through the built in formatter in vscode, so it at least has consistent indenting